deploy/kubeturbo_yamls/turbo_kubeturboDeploy_RBAC_usig_credentials_secret_sample.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  # use this yaml to deploy the kubeturbo pod
+  # Provide a value for the deploy/pod name: and update namespace if needed
+  name: kubeturbo
+  namespace: turbo
+  labels:
+    app: kubeturbo
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        kubeturbo.io/monitored: "false"
+      labels:
+        app: kubeturbo
+    spec:
+      # Update serviceAccount if needed
+      serviceAccount: turbo-user
+      containers:
+        - name: kubeturbo
+          # Replace the image with desired version. Refer to readme in deploy for more details
+          image: turbonomic/kubeturbo:6.4.0
```

Comment:
Should we change the version to `7.21.0` now?

Reply:
Done!